### PR TITLE
[MINOR][DOCS] Fix minor typos in python example code

### PR DESCRIPTION
**Pull request by silentsokolov**
## What changes were proposed in this pull request?

Fix minor typos python example code in streaming programming guide 
## How was this patch tested?

N/A

### CODE REVIEW

**Comment by srowen** [comment]
OK, can you perhaps quickly search for other instances of the same in Python code? it's worth a skim if you're up for it.

**Comment by srowen** [comment]
Jenkins test this please

**Comment by silentsokolov** [comment]
I looked through all the python code examples, fixed typos and formatting

File: docs/structured-streaming-programming-guide.md [hunk 1]
```diff
@@ -487,11 +487,11 @@ Dataset[Row] csvDF = spark
 spark = SparkSession. ...
 
 # Read text from socket 
-socketDF = spark \
-    .readStream()  \
-    .format("socket") \
-    .option("host", "localhost") \
-    .option("port", 9999) \
+socketDF = spark\
```
Comment (srowen) [inline docs/structured-streaming-programming-guide.md]: Yeah I think it's good to standardize strings on double quotes, unless I'm missing some important difference. There are some other good clear fixes here. The whitespace changes are probably not worth it but I'd leave them in. What about this though, is it really more standard not to precede the continuation with a space?

Comment (silentsokolov) [inline docs/structured-streaming-programming-guide.md]: @srowen, there is no clear definition of it.

https://docs.python.org/2/reference/lexical_analysis.html#explicit-line-joining
https://www.python.org/dev/peps/pep-0008/#maximum-line-length

I use:
```python
# with space
if 1900 < year < 2100 and 1 <= month <= 12:
    return 1

if 1900 < year < 2100 \
    and 1 <= month <= 12:
    return 1

# without space
s = 'Hello world'.lower().split()

s = 'Hello world'.lower()\
    .split()
```

This makes the code more readable.

**Comment by srowen** [comment]
@Stibbons since you're around, do you have any comments on these Python style changes?

File: docs/structured-streaming-programming-guide.md (hunk 1 above)
Comment (Stibbons) [inline docs/structured-streaming-programming-guide.md]: Using simple or double quote have impact on Python, it's up to the developer's will.

Adding a space before the '\' makes sense, even though I prefere the following syntax for RDD filters:
```
socketDF = (spark
            .readStream()
            .format("socket")
            .option("host", "localhost")
            .option("port", 9999))
```

**Comment by Stibbons** [comment]
Look good to me. I think it good to have the space arround kwargs assignation removed to distinguish beween
```
a = 1
```
and
```
afunc(a=1)
```
which is a good habit to take

**Comment by Stibbons** [comment]
For information, autopep8 will do all the space changes automatically

**Comment by silentsokolov** [comment]
I returned the spaces

**Comment by srowen** [comment]
OK, it's looking good to me. I wouldn't mind a set of eyes more familiar with Python to look at it. In particular just want to make sure we're headed towards more standardization in all cases.

**Comment by srowen** [comment]
Jenkins retest this please

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**